### ISSUE TITLE
Support viewing collaborate network images for other projects.

### ISSUE THREAD

**Issue by nkorange** [X-lab2017/github-analysis-report-2019#2]
I guess the image https://github.com/X-lab2017/github-analysis-report-2019/blob/master/static/tidb_network.png is generated by code, so is it possible to provide a guide of how to generate collaborate network images for other projects?

**Comment by frank-zsy** [comment]
Yes, of course, the tools used to generate the image are also open source projects. The report is still in processing right now, we will refer them in the report later.
